app/models/game.rb

```diff
@@ -1,4 +1,63 @@
 class Game < ActiveRecord::Base
   belongs_to :users
   has_many :pieces
+  after_create :initialize_board!
+
+  def initialize_board!
+    make_pawns!
+    make_rooks!
+    make_bishops!
+    make_knights!
+    make_kings!
+    make_queens!
+  end
+
+  def make_pawns!
+    8.times do |n|
+        pieces << Pawn.create(
+            color: 'white', 
+            x_cord: n,
+            y_cord: 1
+            )
+    end
+    #
+    8.times do |n|
+        pieces << Pawn.create(
+            color: 'black', 
+            x_cord: n,
+            y_cord: 6
+            )
+    end
+  end
+
+  def make_rooks!
+    pieces << Rook.create(color: 'white', x_cord: 0, y_cord: 0)
+    pieces << Rook.create(color: 'white', x_cord: 7, y_cord: 0)
```

Comment:
Can you explain what the purpose of the `pieces <<` in each line is? I don't get what this is doing.
As far as adding to the specific game they're being created for -- I believe we just need to add `game_id: id` along with the color and coordinates to tie it to the right game -- game_id is contained in the pieces database.